manifests/c/calibre/calibre/portable/7.26.0.0/calibre.calibre.portable.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: calibre.calibre.portable
+PackageVersion: 7.26.0.0
+InstallerLocale: en-US
+InstallerType: exe
+Scope: user
+InstallerSwitches:
+  Silent: /CalibrePortable/
+  SilentWithProgress: /CalibrePortable/
+  Custom: ""
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/kovidgoyal/calibre/releases/download/v7.26.0/calibre-portable-installer-7.26.0.exe

```

```suggestion
  InstallerUrl: https://download.calibre-ebook.com/7.26.0/calibre-portable-installer-7.26.0.exe
```
I think it's better to use this url, Release on GitHub only keeps binaries for the latest version.
Same hash: https://github.com/DuckDuckStudio/winget-tools/actions/runs/13324364186/job/37214498321#step:9:11